docs/docsite/rst/plugins/vars.rst

```diff
@@ -47,6 +51,10 @@ You can also control vars plugin execution on a per-plugin basis for vars plugin
     [vars_host_group_vars]
     stage = inventory
 
+.. note::
+
+   ``ansible-inventory`` is hardcoded to always execute vars plugins and does not use these configuration options.
```

Comment:
it still uses the enabled  and other config, should we specify that this note only applies to 'execution time' configs